src/component/modal/feil-modal/feil-modal.tsx

```diff
@@ -39,9 +37,11 @@ export function FeilModal(props: FeilmodalProps) {
 		>
 			<Systemtittel className="blokk-xxs">{tittel}</Systemtittel>
 			<Normaltekst className="blokk-s">{beskrivelse}</Normaltekst>
-			<Show if={knappeTekst}>
-				<Knapp onClick={handleRequestClose}>{knappeTekst}</Knapp>
-			</Show>
+			{knappeTekst && (
```

Comment:
En tanke slo meg plutselig: burde vi slutte å bruke `someValue && <SomeComponent>`-syntaksen for conditional rendering? Ref. [pitfallen som er beskrevet i React-docen](https://react.dev/learn/conditional-rendering#logical-and-operator-); dersom `someValue` er et tall så kan man potensielt få en bug der dersom `someValue` er `0` så evalueres uttrykket som falsy men selve verdien (`0`) vil rendres.

Jeg tror det er ytterst sjeldent/usannsynlig at dette noen gang vil skje, og jeg gjør dette hele tiden selv, men kom bare til å tenke på det nå 😅

Reply:
Når jeg når har tenkt litt så får jeg fortsatt lyst til å bruke det, men med det forbehold å være ekstra påpasselig når det er tall involvert 🤔 
Altså sånn som her hvor det er en streng så gjør vi ingen endringer, men hvis det er tall så må vi passe på å ha en ekstra sjekk?